api/python/quilt3-admin/base_client.py

```diff
@@ -0,0 +1,211 @@
+# This is
+# https://github.com/mirumee/ariadne-codegen/blob/5bfd63c5e7e3a8cc5293eb94deee638b7adab98d/ariadne_codegen/client_generators/dependencies/base_client.py
+# modified to use our requests session instead of httpx.
+# pylint: disable=relative-beyond-top-level
+import json
+from typing import IO, Any, Dict, List, Optional, Tuple, TypeVar, cast
+
+import requests
+from pydantic import BaseModel
+from pydantic_core import to_jsonable_python
+
+from quilt3 import session
+
+from .base_model import UNSET, Upload
+from .exceptions import (
+    GraphQLClientGraphQLMultiError,
+    GraphQLClientHttpError,
+    GraphQLClientInvalidResponseError,
+)
+
+Self = TypeVar("Self", bound="BaseClient")
+
+
+class BaseClient:
```

Comment:
> can we import and subclass this instead of rewriting? maybe that would lessen maintenance burden idk

to import it we need to depend on ariadne-codegen and I don't think we want it

> or at least mark the modified parts

I don't think someone will want to modify it soon and just doing diff against upstream version makes more sense IMO